### ISSUE TITLE
Push notification

### ISSUE THREAD

**Issue by rgnarok** [technicalbirdVayuz/volleyball-unite-usa-issue-tracker#228]
Push notifications not coming up

**Comment by technicalbirdVayuz** [comment]
This needs to be configured

**Comment by technicalbirdVayuz** [comment]
Have configured this, It can be checked when we receive the access for Google /Apple Accounts

**Comment by rgnarok** [comment]
Google and apple accounts are there why are these still on hold it is not working

**Comment by technicalbirdVayuz** [comment]
1. Do check the date, when it is being put on hold, We don't have accounts at that point of time
2. Update will be shared once it is done.

**Comment by technicalbirdVayuz** [comment]
Lets have discussion here #308
